StreamChatClient_v3/Controllers/ChannelController_Tests.swift

```diff
@@ -14,14 +14,16 @@ class ChannelController_Tests: StressTestCase {
     var channelId: ChannelId!
     
     var controller: ChannelController!
-    
+    let controllerCallbackQueueID = UUID()
```

Comment:
I think this should rather be `let controllerCallbackQueueID: UUID!` and we should recreate it in `setUp()` so we're sure the value changes with every test.

Reply:
fixed in 99dd5f65f64abfeb06eb1f2e9d16d3242cc1b910